### DISCUSSION TITLE
GIT: allow to display icon for "local branches"

### DISCUSSION THREAD

**Discussion by thePanz** [starship/starship#5450] [Q&A]
Is there the possibility to flag (or add a symbol) for branches with no remote tracking?

Why?
I am porting the terminal style of bash-git-prompt https://github.com/magicmonty/bash-git-prompt to Starship, and such flag is not available on Starship, or I was not able to find it.

Any hints?

**Comment by mickimnet** [comment]
There is a custom module, you could use. See: https://github.com/starship/starship/discussions/1252#discussioncomment-196980
